custom_components/vicare/binary_sensor.py

```diff
@@ -43,6 +67,12 @@ class ViCareBinarySensorEntityDescription(
         device_class=DEVICE_CLASS_POWER,
         value_getter=lambda api: api.getCirculationPumpActive(),
     ),
+    ViCareBinarySensorEntityDescription(
+        key=SENSOR_FROST_PROTECTION_ACTIVE,
+        name="Frost protection active",
+        device_class=DEVICE_CLASS_POWER,
```

Comment:
maybe DEVICE_CLASS_SAFETY?

Reply:
Safety in my view is rather related to alarm systems and ON would be displayed as unsafe which would be misleading in my view. We could also have it as DEVICE_CLASS_RUNNING? Or the same as other pumps as it is actually indicating that a pump is running.

Reply:
Not sure which one is the best, how about the `DEVICE_CLASS_HEAT`? It shows as flame when on. In reality the frost protection means that the circuit pump is active when temperature drops below the set point defined in the heater. Running could be ok as well.

Reply:
same as above.
let's merge and maybe we will need to change it once it was contributed upstream